ee/log/osquerylogs/log_posix.go

```diff
@@ -19,24 +19,26 @@ func (l *OsqueryLogAdapter) runAndLogPs(pidStr string) {
 
 	cmd, err := allowedcmd.Ps(ctx, "-p", pidStr, "-o", "user,pid,ppid,pgid,stat,time,command")
 	if err != nil {
-		level.Debug(l.logger).Log(
-			"msg", "error creating command to run ps on osqueryd pidfile",
+		l.slogger.Log(context.TODO(), slog.LevelDebug,
+			"error creating command to run ps on osqueryd pidfile",
```

Comment:
This whole section should probably be level info. It's about detecting some obscure and weird error

Reply:
updated to log an before we start dumping info about locking process (the info dump is info level or error if we encounter error getting info)